packages/ibx/ibx.0.8.0/opam

```diff
@@ -0,0 +1,31 @@
+opam-version: "1.2"
+name: "ibx"
+version: "0.8.0"
```

Comment:
You may want to regenerate the file with the latest version of `oasis2opam` because [it turned out](https://github.com/ocaml/opam/issues/2243) that `name` and `version` are not desired in the repository `opam` files.